src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.24.0'
+__version__ = '1.24.1'
```

Comment:
```suggestion
__version__ = '1.25.0'
```
I think philosophically this is more of a minor bump given that anyone who had been using this functionality would need to change their behavior at this point.  Not required for approval.

Reply:
I think that's a fair point.

Reply:
The one thing I dislike about GitHub's PR system is that no matter what, if you push a change, it invalidates approvals. It makes sense to a certain degree, of course, but I do wish we could say "Approve even if they make changes".